README.md

```diff
@@ -1,39 +1,166 @@
+WHEN I review the app
+THEN I am presented with movie recommendations and Navbar options to search the Movie database and options to Login/Signup
+
+WHEN I am not logged in and enter a search term in the input field and click the submit button
+THEN I am presented with search results, featuring a movies’s title, an option to view the trailer (if available),an overview of the movie and the poster image
+
+WHEN I click on the Login/Signup menu option
+THEN a modal appears on the screen with a toggle between the option to log in or sign up
+
+WHEN the toggle is set to Signup
+THEN I am presented with three inputs for a username, an email address, and a password, and a signup button
+
+WHEN the toggle is set to Login
+THEN I am presented with two inputs for an email address and a password and login button
+
+WHEN I enter a valid email address and create a password and click on the signup button
+THEN my user account is created and I am logged in to the site
+
+WHEN I enter my account’s email address and password and click on the login button
+THEN I the modal closes and I am logged in to the site
+
+WHEN I am logged in to the site
+THEN in addition to trending recommendations the Navbar menu options include options to search the Movie database, see my saved movies, and Logout
+
+WHEN I am logged in and look at recommendations presented to me 
+THEN I have an option to like or dislike movies presented to me or look at more recommendations
+
+WHEN I am logged in and enter a search term in the input field and click the submit button
+THEN I am presented with several search results, each featuring a movies’s title, an option to view the trailer (if available),an overview of the movie, the poster image, and the option to like or dislike the movie.
+
+WHEN I like a movie
+THEN that movie’s information is saved to my account
+
+WHEN I click on the option to see my saved movies
+THEN I am presented with all of the movies I have saved to my account, each featuring the movies’s title, an overview of the movie, the poster image, and an option to dislike the movie (and to remove it from my account)
+
+WHEN I am both logged in and in the saved movies
+THEN I am presented with an option to signup for the FANTINDER streaming service. On signing up for the streaming service, you can come back to the saved movies page
+
+all of the movies I have saved to my account, each featuring the movies’s title, an overview of the movie, the poster image, and an option to dislike the movie (and to remove it from my account)
+
+WHEN I dislike a movie
+THEN that movie is deleted from my saved list
+
+WHEN I click on the Logout button
+THEN I am logged out of the site and presented with a menu with the options to see movie recommendations, and option to Login/Signup, and an option search for a movie and an input field to search for movies and a submit button  
+
+```
+
+
+
+## Concepts
+
+In this project we applied the following skills:
```

Comment:
Maybe instead of "applied the following skills" we should say "used the following packages"?

Reply:
changes made